_docs/04.3-electrode_boilerplate.markdown

```diff
@@ -6,13 +6,13 @@ toplevel: "Electrode Boilerplate"
 
 ###Maximum Performance Out of the Box
 
-In our [Getting Started](get_started.html) section, we introduced you to our Yeoman Electrode [Generator](https://github.com/electrode-io/generator-electrode) and we built an impressive application out of the built in technologies that the simple `yo electrode` command gave us:
+In our [Getting Started](get_started.html) section, we introduced you to our Yeoman Electrode [Generator](https://github.com/electrode-io/generator-electrode) and we constructed an impressive application with the built in technologies that the simple `yo electrode` command gave us:
 
 `React * Redux * CSS Modules * Electrode-Server * Electrode Confippet * Universal JavaScript Babel * Webapack * Webpack Isomorphic Loader * React-Router * History React-Resolver * Electrode Bundle Analyzer Electrode-Static-Paths * Lodash *  Electrode-Docgen * Bluebird * Gulp * ESLint * Mocha * Enzyme * Travis * Yeoman`
```

Comment:
We need to better this message, maybe in a list or link it to https://electrode-io.github.io/docs/what_is_electrode.html#features